src/foam/nanos/auth/Hours.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'Hours',
+
+  documentation: 'Represents open and closing time',
+
+  constants: [
+    {
+      name: 'SUNDAY',
```

Comment:
Remove trailing , and empty line.

Reply:
The constants should be removed altogether. Removed.